core/lib/zksync_core/src/state_keeper/io/seal_logic.rs

```diff
@@ -409,6 +402,7 @@ impl MiniblockSealCommand {
 
         let progress =
             MINIBLOCK_METRICS.start(MiniblockSealStage::ExtractContractsDeployed, is_fictive);
+        let unique_updates = HashMap::new(); // FIXME
```

Comment:
`ExtractContractsDeployed` is measured in single-digit microseconds now, so we can ditch counting the value per unit - feel free to get rid of `unique_updates` then (it's only used for this right?)